File: 485-max-consecutive-ones/max-consecutive-ones.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function(nums) {
    let curr=0;
    let res=0;
    for(let item of nums){
        if(item==1){
            curr++;
            res=Math.max(res,curr);
        }
        else{
            curr=0;
        }

    }
    return res;
};

File: 485-max-consecutive-ones/max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        max1=0;
        count=0;
        for num in nums:
            if num==1:
                count+=1;
                max1=max(max1,count);
            else:
                count=0;
        return max1;
        